tests/components/abode/test_light.py

```diff
@@ -0,0 +1,119 @@
+"""Tests for the Abode light device."""
+from unittest.mock import patch
+
+from homeassistant.components.abode import ATTR_DEVICE_ID
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP,
+    ATTR_RGB_COLOR,
+    DOMAIN as LIGHT_DOMAIN,
+)
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    ATTR_FRIENDLY_NAME,
+    ATTR_SUPPORTED_FEATURES,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_ON,
+)
+
+from .common import setup_platform
+
+DEVICE_ID = "light.living_room_lamp"
+
+
+async def test_entity_registry(hass):
+    """Tests that the devices are registered in the entity registry."""
+    await setup_platform(hass, LIGHT_DOMAIN)
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
+
+    entry = entity_registry.async_get(DEVICE_ID)
+    assert entry.unique_id == "741385f4388b2637df4c6b398fe50581"
+
+
+async def test_attributes(hass):
+    """Test the light attributes are correct."""
+    await setup_platform(hass, LIGHT_DOMAIN)
+
+    state = hass.states.get(DEVICE_ID)
+    assert state.state == STATE_ON
+    assert state.attributes.get(ATTR_BRIGHTNESS) == 204
+    assert state.attributes.get(ATTR_RGB_COLOR) == (0, 63, 255)
+    assert state.attributes.get(ATTR_COLOR_TEMP) == 280
+    assert state.attributes.get(ATTR_DEVICE_ID) == "ZB:db5b1a"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5184102)